### ISSUE TITLE
error handling for non-github repos

### ISSUE THREAD

**Issue by HP4k1h5** [ldelossa/gh.nvim#1]
Great project. Thank you. Looking forward to using this tool.

I noticed that the erroring around non-github repos doesn't provide much help. It might seem obvious that PR's are not a feature of git but rather of github, but it's easy to forget or simply not know where git and gh differ.

### proposal
A clearer error set of error messages when the repo is not hosted by github would be welcome. 

Happy to take this on if you are accepting contributors. Cheers and thanks again.

**Comment by ldelossa** [comment]
Definitely! I didnt even think this case through. Contributions more then welcome.
